Automattic/wp-calypso
# Add new Earn features under the Free plan

Related to pdgrnI-2xq-p2#comment-4128 and code-D115905.

## Proposed Changes

* Add "Payment button" feature under the Free plan.
* Add "Pay with PayPal" feature in the comparison grid only (it is available on Premium plan and upwards).

<img width="1343" alt="Screenshot 2023-07-13 at 3 50 41 PM" src="https://github.com/Automattic/wp-calypso/assets/1269602/2116acd7-0835-4e15-bbd7-3a6f28922727">

<img width="1280" alt="Screenshot 2023-07-13 at 3 51 00 PM" src="https://github.com/Automattic/wp-calypso/assets/1269602/fee7e8ae-d03b-4f9b-a6f5-085e7a5c5d50">


## Testing Instructions



* Enter the signup flow at `/start`, and in `/start/plans` verify that the features mentioned in the description are visible, both in the main grid and the comparison grid.
* Feature should be visible in both monthly and annual tabs. 
* Tooltip content should be visible in both main grid and comparison grid.
* Non-en will show untranslated content, `String freeze` label is added.
* Verify both desktop and mobile viewports.
* Verify changes are visible in signup flow plans step and in Calypso /plans page.
## Pre-merge Checklist
